RULES.md

```diff
@@ -275,3 +275,27 @@ Description: Checks that that the ingress permissions on an EC2 Security Group a
 
 Trigger Type: ```Change Triggered```
 Required Parameter: ```None```
+
+### 32. Check that no EC2 is publicly accessible except 80 and 443.
```

Comment:
Changes added in RULES.md https://github.com/PriceBoardIn/aws-config-rules/commit/8c06c87a0922742080fd3048ce37fbaf9a98ee60